text/0232-maturity-of-otel.md

```diff
@@ -0,0 +1,151 @@
+
+#### Stable
+
+The component is ready for general availability. Bugs and performance problems should be reported, and there's an expectation that the component owners will work on them. Breaking changes, including configuration options and the component's output, are not expected to happen without prior notice unless under special circumstances.
+
+#### Deprecated
+
+Development of this component is halted, and no further support will be provided. No new versions are planned, and the component might be removed from its included distributions. Note that new issues will likely not be worked on except for critical security issues. Components that are included in distributions are expected to exist for at least two minor releases or six months, whichever happens later. They also MUST communicate in which version they will be removed.
+
+#### Unmaintained
+
+A component identified as unmaintained does not have an active code owner. Such components may have never been assigned a code owner, or a previously active code owner has not responded to requests for feedback within 6 weeks of being contacted. Issues and pull requests for unmaintained components SHOULD be labeled as such. After 6 months of being unmaintained, these components MAY be deprecated. Unmaintained components are actively seeking contributors to become code owners.
+
+### Specification requirements
+
+The specification should declare all requirements for a specified component to be declared stable. When implementations are still missing features required by the specification, they can be at most Alpha status.
+
+### Tier-1 SIGs
+
+This OTEP recommends the following SIGs to be included as part of the Tier-1 SIGs for OpenTelemetry:
+
+* Spec General
+* Semantic Conventions (WG)
+* Java
+* Java auto-instrumentation
+* Go
+* .Net
+* JavaScript
+
+Once this OTEP is approved, the TC and GC members will vote on the individual SIGs from the list above and are free to make further suggestions. A final list of Tier-1 SIGs shall be in the "community" repository.
+
+### Tier-1 Components
+
+When discussing the components' maturity, we often debate what should be included in the evaluation criteria. To help clarify that, we introduce the concept of "Tier-1 SIGs" and "Tier-1 Components". Tier-1 Components are critical for a specific SIG: only when all Tier-1 Components are stable can a project be evaluated as stable. For instance, the "OTLP receiver" for the OpenTelemetry Collector could potentially be a "Tier-1 Component". Only after this component is declared stable may the Collector be declared stable, provided all other Tier-1 Components are also stable.
```

Comment:
> Only after this component is declared stable may the Collector be declared stable, provided all other Tier-1 Components are also stable.

Assuming Tier-1 components are also considered dependencies, this requirement is a partial version of a previous requirement noted above,

> Only once all dependencies are marked as "stable" MAY a component be marked as "stable"

So either the Tier-1 requirement noted here is redundant or something else is wrong :)

Reply:
It is redundant. I was just being extra clear that the same requirements apply downstream.

Reply:
Alright! You can mark this as solved then, just found it confusing

Reply:
Clearer now indeed, thanks!